### PR TITLE
Update release-notes.html

### PR DESCRIPTION
**Pull request by tayronemachadozup**
beagle 1.8 release notes

### CODE REVIEW

File: content/pt/release-notes.html [hunk 1]
```diff
@@ -1,173 +1,179 @@
...
+    <div class="container">
+        <div class="beagle_brand">
+            <a href="https://usebeagle.io"><img src="https://docs.usebeagle.io/shared/Beagle.svg" /></a>
+        </div>
+        <div class="background-image">
+            <img src='https://docs.usebeagle.io/shared/Release.svg'>
+        </div>
+         <label class='beagle__version__text'><a target="_blank" class="content__list__link" href="https://github.com/ZupIT/beagle/releases/tag/1.8.0">Versão 1.8 </a></label>
+        <div class="block-list">
+            <label class="titleList">New functionalities</label>
```
Comment (matheusribeirozup) [inline content/pt/release-notes.html:136]: Esse release notes está no diretório `pt`, não deveria ser todo escrito em português?

Comment (carlossteinzup) [inline content/pt/release-notes.html:136]:
```suggestion
            <label class="titleList">Novas funcionalidades</label>
```

File: content/pt/release-notes.html [hunk 2]
```diff
@@ -1,173 +1,179 @@
...
+            <div>
+                <ul>
+                    
+            <li class='content__list__item'>
+                <img src="../assets/check.svg" class="iconSvg"/><a target="_blank" class="content__list__link" href="https://docs.usebeagle.io/v1.8/api/components/layout/gridview/">GridView component</a>
```
Comment (carlossteinzup) [inline content/pt/release-notes.html:141]:
```suggestion
                <img src="../assets/check.svg" class="iconSvg"/><a target="_blank" class="content__list__link" href="https://docs.usebeagle.io/v1.8/api/components/layout/gridview/">Componente: GridView</a>
```

File: content/pt/release-notes.html [hunk 3]
```diff
@@ -1,173 +1,179 @@
...
+            </li>
+    
+            <li class='content__list__item'>
+                <img src="../assets/check.svg" class="iconSvg"/><a target="_blank" class="content__list__link" href="https://docs.usebeagle.io/v1.7/api/components/layout/listview/">ListView with multiples templates</a>
```
Comment (carlossteinzup) [inline content/pt/release-notes.html:145]:
```suggestion
                <img src="../assets/check.svg" class="iconSvg"/><a target="_blank" class="content__list__link" href="https://docs.usebeagle.io/v1.7/api/components/layout/listview/">Componente: ListView com multiplos templates</a>
```

File: content/pt/release-notes.html [hunk 4]
```diff
@@ -1,173 +1,179 @@
...
+            </li>
+    
+            <li class='content__list__item'>
+                <img src="../assets/check.svg" class="iconSvg"/><a target="_blank" class="content__list__link" href="https://docs.usebeagle.io/v1.8/api/actions/addchildren/">Add support to bind in field value in action add children </a>
```
Comment (carlossteinzup) [inline content/pt/release-notes.html:149]:
```suggestion
                <img src="../assets/check.svg" class="iconSvg"/><a target="_blank" class="content__list__link" href="https://docs.usebeagle.io/v1.8/api/actions/addchildren/">Adiciona suporte bind para o campo "value" na ação "add children"</a>
```

File: content/pt/release-notes.html [hunk 5]
```diff
@@ -1,173 +1,179 @@
...
+            </li>
+    
+                </ul>
             </div>
-    </body>
-</html>
\ No newline at end of file
+            <label class="titleList">Impacts in iOS/Android/Web <img class='plataform__icon' src='https://docs.usebeagle.io/shared/IOS.svg'/> <img class='plataform__icon' src='https://docs.usebeagle.io/shared/Android.svg'/> <img class='plataform__icon' src='https://docs.usebeagle.io/shared/Web.svg'/></label>
```
Comment (carlossteinzup) [inline content/pt/release-notes.html:154]:
```suggestion
            <label class="titleList">Impactos no iOS/Android/Web <img class='plataform__icon' src='https://docs.usebeagle.io/shared/IOS.svg'/> <img class='plataform__icon' src='https://docs.usebeagle.io/shared/Android.svg'/> <img class='plataform__icon' src='https://docs.usebeagle.io/shared/Web.svg'/></label>
```

File: content/pt/release-notes.html [hunk 6]
```diff
@@ -1,173 +1,179 @@
...
+            <div>
+                <ul>
+                    
+            <li class='content__list__item'>
+                <img src="../assets/check.svg" class="iconSvg"/><a target="_blank" class="content__list__link" href="https://github.com/ZupIT/beagle/releases/tag/1.8.0">Bugs fixed</a>
```
Comment (carlossteinzup) [inline content/pt/release-notes.html:159]:
```suggestion
                <img src="../assets/check.svg" class="iconSvg"/><a target="_blank" class="content__list__link" href="https://github.com/ZupIT/beagle/releases/tag/1.8.0">Bugs corrigidos</a>
```

File: content/pt/release-notes.html [hunk 7]
```diff
@@ -1,173 +1,179 @@
...
+    
+                </ul>
+            </div>
+        </div>
+        <div class="background-image">
+            <img src="https://docs.usebeagle.io/shared/People.svg">
+        </div>
+        <footer>
+            <p>Thanks.</p>
+            <p>Team Beagle</p>
```
Comment (carlossteinzup) [inline content/pt/release-notes.html:170]:
```suggestion
            <p>Obrigado!</p>
            <p>Time Beagle</p>
```

**Review by dantes-git — CHANGES_REQUESTED** [review]
The correct thing is to update the release notes of the _content/en_ folder in English and the _content/pt_ folder in Portuguese. I also believe that there is a procedure to generate these release notes easier, talk to @luismaximozup to check if this really exists.

**Pull request closed without merging**